server/src/test/java/io/crate/integrationtests/TransportSQLActionClassLifecycleTest.java

```diff
@@ -590,7 +567,7 @@ public void selectWhereEqualCurrentTimestamp() throws Exception {
         SQLResponse response = execute("select * from sys.cluster where current_timestamp = current_timestamp");
         assertThat(response.rowCount(), is(1L));
 
-        SQLResponse newResponse = execute("select * from sys.cluster where current_timestamp > current_timestamp");
+        SQLResponse newResponse = execute("select * from sys.cluster where current_timestamp < current_timestamp");
```

Comment:
Why the change here? Semantics should still be the same?

Reply:
yes, they are the same